application/pm.py

```diff
@@ -149,8 +149,12 @@ def create_project_dir(self):
             copy(self.videopath, video_dest)
 
             progress_msg.setText("Uploading video file...")
-            identifier = api.uploadVideo(self.videopath)
-            update_config_with_sections(get_config_path(), 'info', 'identifier', identifier)
+            try:
+                identifier = api.uploadVideo(self.videopath)
+                update_config_with_sections(get_config_path(), 'info', 'identifier', identifier)
+            except Exception as e:
+                print("Failed to upload the Video")
+                print(str(e))
```

Comment:
Probably abort here. If actual error handling isn't within the scope of this PR, ignore this.